projects/core/src/organization/store/selectors/budget.selector.ts

```diff
@@ -0,0 +1,42 @@
+import { createSelector, MemoizedSelector } from '@ngrx/store';
+import {
+  StateWithOrganization,
+  OrganizationState,
+} from '../organization-state';
+import { getOrganizationState } from './feature.selector';
+import { Budget } from '../../../model/budget.model';
+import { EntityLoaderState } from '../../../state/utils/entity-loader/index';
+import { LoaderState } from '../../../state/utils/loader/loader-state';
+import { entityStateSelector, entityValueSelector } from '../../../state/utils/entity-loader/entity-loader.selectors';
+
+export const getBudgetManagmentState: MemoizedSelector<
+  StateWithOrganization,
+  EntityLoaderState<Budget>
+> = createSelector(
+  getOrganizationState,
+  (state: OrganizationState) => state.budgetManagment
+);
+
+export const getBudgetsState: MemoizedSelector<
```

Comment:
I don't see a big advantage from exposing an object with all EntityLoaderStates.

Reply:
it is used in facade, instead of combining a lot of smaller selectors